packages/components/src/FilterBar/_docs/FilterBar.mdx

```diff
@@ -189,6 +219,12 @@ A callback (`(values: Partial<ValuesMap>) => void`) for when the active `values`
 
 ## Examples
 
+### Dependent filters
+
+Example where `Topping` and `Drank` are usable only if `Flavour` has a value:
```

Comment:
It's a bit hard to look at the live example code and find the exact thing that's driving the example and learn how to apply it, eg. I might have existing config and jump straight to this example hoping to retrofit.

So I think it would be nice to have a succinct code snippet here that highlights the `isUsableWhen` config, maybe just the filters array.